### PR TITLE
rocksdanister.LivelyWallpaper version 1.6.0.4

### PR DESCRIPTION
**Pull request by rocksdanister**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16203)

### CODE REVIEW

**Comment by KaranKad** [comment]
It prompts me for user or system install
image

**Comment by rocksdanister** [comment]
I noticed it too, when I tried with admin terminal winget --silent flag install succeeded.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-pkgs/discussions/16233

**Comment by KaranKad** [comment]
I think winget by default passes `SilentWithProgress` switch for all applications, here you have entered proper switches for the application but for `Silent` switch only, as winget uses `SilentWithProgress` by default and that field was not mentioned, winget passed default switch for inno setups for `SilentWithProgress` which is just `/SILENT` hence failed result.
Solution for this would be adding `SilentWithProgress` switch with `/SILENT /SUPPRESSMSGBOXES /NORESTART /SP- /CURRENTUSER /NOAUTOLAUNCH /NODEPENDENCIES`


Here I changed `Silent` switch to `Custom`, custom switch will be forced to the installer, so winget forced `/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /CURRENTUSER /NOAUTOLAUNCH /NODEPENDENCIES` switch to the installer and it installed successfully.
image

**Comment by rocksdanister** [comment]
Looks like missing `SilentWithProgress` was the problem, thanks for the help.

File: manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: rocksdanister.LivelyWallpaper
+PackageVersion: 1.6.0.4
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/rocksdanister/lively/releases/download/v1.6.0.4/lively_setup_x86_full_v1604.exe
+  InstallerSha256: 706F9E7176E1DB242B79583AEEEDCD3BAD199E7D05DA29775A678891601C0238
+  InstallerSwitches:
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /CURRENTUSER /NOAUTOLAUNCH /NODEPENDENCIES
+    SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART /SP- /CURRENTUSER /NOAUTOLAUNCH /NODEPENDENCIES
```
Comment (OfficialEsco) [inline manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:12]:
```suggestion
  InstallerSwitches:
    Custom:  /NORESTART /NOAUTOLAUNCH /NODEPENDENCIES /CURRENTUSER 
    Scope: user
- Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/rocksdanister/lively/releases/download/v1.6.0.4/lively_setup_x86_full_v1604.exe
  InstallerSha256: 706F9E7176E1DB242B79583AEEEDCD3BAD199E7D05DA29775A678891601C0238
  InstallerSwitches:
    Custom: /NORESTART /NOAUTOLAUNCH /NODEPENDENCIES /ALLUSERS
  Scope: machine
```

File: manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: rocksdanister.LivelyWallpaper
+PackageVersion: 1.6.0.4
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]:
```suggestion
- Architecture: neutral
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Not trying to be evil but imma need these changes for compatibility 👀 Also for cleanness sake c:
Also for some reason with the /CURRENTUSER switch the application turns into x64 so i'll take a guess that its neutral.

File: manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml (hunk 2 above)
Comment (rocksdanister) [reply manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]: That's impossible.. its x86 build.

Comment (OfficialEsco) [reply manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]: Didn't notice you were the dev, ignore that one, I'll report a bug if my eyes wasn't going sideways 🙄

Comment (OfficialEsco) [reply manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]: image
SEE I'M NOT CRAZY! >:C

**Review by OfficialEsco — APPROVED** [review]

File: manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml (hunk 2 above)
Comment (rocksdanister) [reply manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]: Is it some bug? taskmgr shows livelywpf.exe as 32bit right? 🤔

Comment (OfficialEsco) [reply manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml:6]: > Is it some bug with the tool.. taskmgr shows livelywpf.exe as 32bit right? 🤔

Indeed it does, might be a issue how Windows sees x86 user installed programs then

**Pull request merged**